### DISCUSSION TITLE
2024/09/01/add-multi-remote-on-git-repo/

### DISCUSSION THREAD

**Discussion by giscus** [licyk/licyk.github.io#20] [Comment]
# 2024/09/01/add-multi-remote-on-git-repo/

一个简单的方法为 Git 仓库添加多个源。

https://licyk.netlify.app/2024/09/01/add-multi-remote-on-git-repo/

**Comment by kymi1723** [comment]
你做的好，做得好啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊
